app/models/event.rb

```diff
@@ -125,6 +125,17 @@ def human_game_mode
     self.class.human_game_mode game_mode
   end
 
+  def fitting_teams(user)
+    fitting_teams = user.owned_teams.multiplayer
+    fitting_teams.each do |team|
+      team_member_count = team.members.count
+      if not (min_players_per_team <= team_member_count && max_players_per_team >= team_member_count)
```

Comment:
Move this into a separate method like `is_eligible?(team)`,  where you set line 131 and return the condition of 132 (and let's negate the condition so we can get rid of the `not`) 😄.